go/pkg/filemetadata/filemetadata_test.go

```diff
@@ -41,6 +41,7 @@ func TestComputeFilesNoXattr(t *testing.T) {
 	for _, tc := range tests {
 		t.Run(tc.name, func(t *testing.T) {
 			before := time.Now().Truncate(time.Second)
+			time.Sleep(5 * time.Second)
```

Comment:
fyi @WizardStan This is fixing a flaky test (mtimes have a margin of error and are rounded to the nearest second, so we usually have a sleep in the tests that test for timestamp just before we create the file).